cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -31,6 +31,14 @@ def fill(self, value):
             return True
         return False
 
+    def click(self):
```

Comment:
Required:
This is not way to go for this. I would inherit `ClickableMixin` form `wt.core`. It will provide you `click()` method.

so, try like this.
```
class EntryPoint(Input, ClickableMixin):
```

Let me, its work or not.

Reply:
Yes, It is providing ```click()``` method.
Thanks :)